src/containers/teamPage/index.tsx

```diff
@@ -0,0 +1,268 @@
+    user_id: 2,
+    username: 'jackblanc',
+    team_role: TeamRole.MEMBER,
+  },
+  {
+    user_id: 3,
+    username: 'atreecounter',
+    team_role: TeamRole.NONE,
+  },
+  {
+    user_id: 4,
+    username: 'speakerofthetrees',
+    team_role: TeamRole.PENDING,
+  },
+  {
+    user_id: 5,
+    username: 'vrushalitarte01',
+    team_role: TeamRole.MEMBER,
+  },
+  {
+    user_id: 6,
+    username: 'willmt80',
+    team_role: TeamRole.MEMBER,
+  },
+  {
+    user_id: 7,
+    username: 'apradhan12',
+    team_role: TeamRole.MEMBER,
+  },
+  {
+    user_id: 8,
+    username: 'SofieDunt',
+    team_role: TeamRole.MEMBER,
+  },
+];
+
+const sampleGoalData: GoalProps[] = [
+  {
+    id: 1,
+    goal: 50,
+    progress: 5,
+    start_date: new Date(2021, 0, 1),
+    complete_by: new Date(2022, 0, 1),
+    completion_date: null,
+  },
+  {
+    id: 2,
+    goal: 500,
+    progress: 71,
+    start_date: new Date(2021, 2, 20),
+    complete_by: new Date(2021, 4, 21),
+    completion_date: null,
+  },
+  {
+    id: 3,
+    goal: 10,
+    progress: 10,
+    start_date: new Date(2020, 5, 10),
+    complete_by: new Date(2020, 7, 10),
+    completion_date: new Date(2020, 6, 13),
+  },
+];
+
+const sampleTeamData: TeamProps = {
+  id: 1,
+  name: 'Code4Community',
+  bio: 'Short description about how cool of a team Code4Community is!',
+  members: sampleMemberData,
+  goals: sampleGoalData,
+};
+
+const TeamPage: React.FC = () => {
+  const onClick = () => {
+    // TODO: join team
+  };
+
+  return (
+    <TeamContainer>
+      <StyledLinkButton to={Routes.LOGIN}>
+        {`<`} Return to Teams
+      </StyledLinkButton>
+      <TeamHeaderContainer>
+        <PageHeaderContainer>
+          <PageHeader
+            pageTitle={sampleTeamData.name}
+            pageSubtitle={sampleTeamData.bio}
+          />
+        </PageHeaderContainer>
+        <JoinButton type="primary" onClick={onClick}>
+          Join Team
+        </JoinButton>
+      </TeamHeaderContainer>
+
+      <CenterDiv>
+        <MemberContainer>
+          <SectionHeader>TEAM MEMBERS</SectionHeader>
+          <StyledMemberList
+            dataSource={sampleTeamData.members}
+            itemLayout="vertical"
+          >
+            {sampleTeamData.members.map((member) => {
+              if (
+                member.team_role === TeamRole.LEADER ||
+                member.team_role === TeamRole.MEMBER
+              ) {
+                return (
+                  <TeamMember
+                    id={member.user_id}
+                    team_role={member.team_role}
+                    username={member.username}
+                  />
+                );
+              }
+              return <></>;
+            })}
+          </StyledMemberList>
+        </MemberContainer>
+        <GoalContainer>
+          <SectionHeader>TEAM GOALS</SectionHeader>
+          <LeaderboardCollapse
+            bordered={false}
+            key={1}
+            defaultActiveKey={sampleTeamData.goals[0].id}
```

Comment:
❓ Confused what this is doing. Is key a prop we have to store in React State and control, or does the component manage that internally? If it's managed internally, why the `key={1}`?

Reply:
Forgot to delete the `key={1}` - was a temporary workaround 😂 . `defaultActiveKey` expands the first goal so you can see the stats upon opening the page (instead of all panels being closed)